collector/collectors.go

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) 2019 Hudson River Trading LLC
```

Comment:
I was trying to comply with [this part](https://github.com/manahl/prometheus-flashblade-exporter/blob/c4345dcedb20b3a878c90dc3d2c00696ff4cc490/LICENSE#L294) of the GPLv2 license, which states that every file contributed to should at least contain the copyright header.

Reply:
Thanks for pointing out the line of the GPLv2 license - interesting stuff. Having read it, it seems to be that this is how you apply the license to your new programs, as it's under the heading "How to Apply These Terms to Your New Programs", rather than for contributions.

Hopefully it's clear already that this code can be shared with GPL v2 because of the LICENSE file at the top of the repository, but I can certainly explicitly state the license in more places under the recommendation in the license.

As it stands, I think the copyright notices here ought to be removed. Thanks!

Reply:
In retrospect, I didn't convey my intent properly, and probably came off as pedantic. I apologize. I didn't mean it. I also made the mistake of not copying the default header along with the copyright. This is how it looks now: 
```//
// This program is free software; you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation; either version 2 of the License, or
// (at your option) any later version.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
// GNU General Public License for more details.
//
// You should have received a copy of the GNU General Public License along
// with this program; if not, write to the Free Software Foundation, Inc.,
// 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
// 
// <your copyright statement>
//
// Copyright (c) 2019 Hudson River Trading LLC
// All rights reserved.
//
```
Adding these copyright statements was recommended to me by my company's legal dept to enforce the GPLv2 licence. Of course your own copyright statement should precede mine.

But if you'd rather not have them in individual files, let me know. I'll go talk to my legal dept about this again.

Reply:
I got in touch with someone who knows about software licensing and this sort of thing (i.e. not me). Apparently you're right that we should have the license on every file: this isn't something we've been doing, so thanks for bringing it to my attention!

As a suggestion, how about we put the minimal GPL recommendation at the top of each file ("each file should have at least the "copyright" line and a pointer to where the full notice is found."):
```
Copyright (C) 2019 by the authors in the project README.md
See the full license in the project LICENSE file.
```

and refer to all authors' copyright with a section in the README.md:
```
## Authors
Prometheus FlashBlade Exporter has been under development at Man Group since 2019 and welcomes contributions.

* [Michael Captain](https://github.com/macaptain), Man Group
* <However you want your name or company or both to appear here>
```

Would that be alright?

Reply:
Sure! That works. I have updated all the files with the headers + the Readme.md as per your suggestion. Let me know if that works.